akka-remote/src/main/scala/akka/remote/transport/netty/NettyTransport.scala

```diff
@@ -361,7 +361,7 @@ class NettyTransport(val settings: NettyTransportSettings, val system: ExtendedA
     new ChannelPipelineFactory {
       override def getPipeline: ChannelPipeline = {
         val pipeline = newPipeline
-        if (EnableSsl) pipeline.addFirst("SslHandler", sslHandler(isClient = true))
+        if (EnableSsl) pipeline.addFirst("SslHandler", sslHandler(Option(remoteAddress), isClient = true))
```

Comment:
Is `remoteAddress` nullable here? Doesn't *seem* so, in that case `Some(remoteAddress)` might be clearer.